apps/docs/pages/guides/auth/auth-helpers/nextjs-server-components.mdx

```diff
@@ -254,7 +258,7 @@ export default async function RootLayout({ children }) {
 }
 ```
 
-We don't want Next.js to cache this `session` value, so we need to export a `revalidate` value of `0`.
+We don't want Next.js to cache this `session` value, so we need to export a `revalidate` value of `0` in the layout file.
```

Comment:
Although I was able to infer by looking down further into this doc, it wasn't obvious whether this change still pertained to the layout file